### DISCUSSION TITLE
When agent mode would be moved to stable?

### DISCUSSION THREAD

**Discussion by BlueBlue-Lee** [prometheus/prometheus#10979] [General]
Prometheus agent mode is now an experimental feature, and I want to use this feature on our production environment. So I want to know is there any plan when or which release version  'agent mode feature'  can graduate to stable.

**Comment by roidelapluie** [comment]
There are two points I have in mind before we can mark the agent mode as stable:

- https://github.com/prometheus/prometheus/issues/10431
- https://github.com/prometheus/prometheus/pull/10231

I expect to move on on the second one quite soon ; after the 2.37 LTS is out. Maybe it will help wuth the first issue.

cc @rfratto

**Comment by w0aw** [comment]
any update on this?
